### ISSUE TITLE
"Clear" completed tasks

### ISSUE THREAD

**Issue by fordaj** [kaileywaal/kozukai#2]
When I complete a task, it stays in the same spot with no indication (other than populating the history log) of completion. What do you think of clearing/graying out/italicizing/somehow signifying that I've completed it? Maybe tasks need to be configured to repeat X times daily, then get crossed out and moved to the bottom?
<img width="1172" alt="image" src="https://github.com/kaileywaal/kozukai/assets/26288298/125a7b05-3eb1-4ddd-b032-d97aa054c784">

**Comment by kaileywaal** [comment]
Yes! The plan is to eventually have 3 types of tasks -- one-off tasks that will disappear once completed, tasks that you do once per day max (or maybe a configurable amount of times per day), and habits that you can repeat multiple times per day :)
